src/components/FormDatePicker.vue

```diff
@@ -162,6 +159,12 @@ export default {
             ? this.validator.errors.get(this.name)
             : [];
       }
+    },
+    value(valuee) {
+      if (!!valuee && valuee.length > 0) {
+        const date = moment(valuee, checkFormats, true);
```

Comment:
What happens if the `value` is not a valid date? In the previous code you were using `this.parsingInputDate()` that checks if the date is valid.